py_lesson5_flask/flask5.py

```diff
@@ -0,0 +1,72 @@
+# Необходимо создать API для управления списком задач. Каждая задача должна содержать заголовок и описание. Для каждой задачи должна быть возможность указать статус (выполнена/не выполнена).
+#
+# API должен содержать следующие конечные точки:
+# — GET /tasks — возвращает список всех задач.
+# — GET /tasks/{id} — возвращает задачу с указанным идентификатором.
+# — POST /tasks — добавляет новую задачу.
+# — PUT /tasks/{id} — обновляет задачу с указанным идентификатором.
+# — DELETE /tasks/{id} — удаляет задачу с указанным идентификатором.
+#
+# Для каждой конечной точки необходимо проводить валидацию данных запроса и ответа. Для этого использовать библиотеку Pydantic.
+
+from fastapi import FastAPI, HTTPException
+from pydantic import BaseModel
+from typing import Optional, Dict
+
+app = FastAPI()
+
+
+# Модель для описания задачи
+class Task(BaseModel):
+    title: str
+    description: str
+    status: bool
+
+
+# База данных для хранения задач
+db: Dict[int, Task] = {}
+
+
+# Генератор идентификатора задачи
+def get_next_task_id() -> int:
+    return max(db.keys(), default=0) + 1
```

Comment:
Функция max по умолчанию будет искать максимальное среди ключей.
```suggestion
    return max(db, default=0) + 1
```